doc/api/errors.md

```diff
@@ -222,7 +222,7 @@ myObject.stack;  // similar to `new Error().stack`
 ```
 
 The first line of the trace, instead of being prefixed with `ErrorType:
-message`, will be the result of calling `targetObject.toString()`.
+message`, will be formatted as `Error: <targetObject message>`.
```

Comment:
I achieve `myObject.toString` in sample code, but the output doesn't change. So, I think `Error.captureStackTrace` don't use `target.toString()`. 
I run sample code in chrome, and get same output. does this feature come from v8?